.travis.yml

```diff
@@ -185,7 +218,7 @@ script:
         python tools/suppress_output.py pip wheel --no-build-isolation .
         pip install --no-cache-dir scipy*.whl
         USE_WHEEL_BUILD="--no-build"
-    elif [ "${USE_SDIST}" == "1" ]; then
+    elif [ "${USE_SDIST}" == "1" -a "${TRAVIS_CPU_ARCH}" != "arm64"  ]; then
```

Comment:
Here the logic gets messed up. In the ARM64 build job definition `USE_SDIST=1` is set, however here it's being bypassed. There are two options here:
1. Remove `USE_SDIST=1` and undo this change, or
2. just undo this change 

I'd prefer (2).